ankerctl.py

```diff
@@ -165,24 +194,98 @@ def mqtt_gcode(env):
         else:
             log.error("No response from printer")
 
+
 @main.group("pppp", help="Low-level pppp api access")
 def pppp(): pass
 
+
 @pppp.command("lan-search")
 @pass_env
 def pppp_lan_search(env):
+    """
+    Attempt to find available printers on local LAN.
+
+    Works by broadcasting a LAN_SEARCH packet, and waiting for a reply.
+    """
     api = AnkerPPPPApi.open_broadcast(timeout=1.0)
     try:
-        resp = api.req(PktLanSearch())
+        api.send(PktLanSearch())
+        resp = api.recv()
     except TimeoutError:
         log.error("No printers responded within timeout. Are you connected to the same network as the printer?")
     else:
         if isinstance(resp, libflagship.pppp.PktPunchPkt):
             log.info(f"Printer [{str(resp.duid)}] is online")
 
+
+@pppp.command("print-file")
+@click.argument("file", required=True, type=click.File("rb"), metavar="<file>")
+@click.option("--no-act", "-n", is_flag=True, help="Upload only (do not start printing)")
```

Comment:
`--no-print` or `--dry-run` may be more intuitive semantics over `--no-act`

Reply:
How do you feel about the naming in the newest version?